src/ray/core_worker/transport/direct_actor_transport.cc

```diff
@@ -530,6 +530,16 @@ void CoreWorkerDirectTaskReceiver::HandleTask(
     send_reply_callback(Status::Invalid("client cancelled stale rpc"), nullptr, nullptr);
   };
 
+  auto steal_callback = [this, task_spec,
+                         reply](rpc::SendReplyCallback send_reply_callback) {
+    RAY_LOG(DEBUG) << "Task " << task_spec.TaskId() << " was stolen from "
+                   << worker_context_.GetWorkerID()
+                   << "'s non_actor_task_queue_! Setting reply->set_task_stolen(true)!";
+    // task stolen. respond accordingly
```

Comment:
Can you make sure to use full sentences (periods and capitalization) for comments? Probably, this comment is not necessary.